docs/teams.md

```diff
@@ -1 +1,125 @@
+
+```hcl
+  # EKS Application Teams
+
+  application_teams = {
+    # First Team
+    team-blue = {
+      "labels" = {
+        "appName"     = "example",
+        "projectName" = "example",
+        "environment" = "example",
+        "domain"      = "example",
+        "uuid"        = "example",
+      }
+      "quota" = {
+        "requests.cpu"    = "1000m",
+        "requests.memory" = "4Gi",
+        "limits.cpu"      = "2000m",
+        "limits.memory"   = "8Gi",
+        "pods"            = "10",
+        "secrets"         = "10",
+        "services"        = "10"
+      }
+      manifests_dir = "./manifests"
+      # Belows are examples of IAM users and roles
+      users = [
+        "arn:aws:iam::123456789012:user/blue-team-user",
+        "arn:aws:iam::123456789012:role/blue-team-sso-iam-role"
+      ]
+    }
+
+    # Second Team
+    team-red = {
+      "labels" = {
+        "appName"     = "example2",
+        "projectName" = "example2",
+      }
+      "quota" = {
+        "requests.cpu"    = "2000m",
+        "requests.memory" = "8Gi",
+        "limits.cpu"      = "4000m",
+        "limits.memory"   = "16Gi",
+        "pods"            = "20",
+        "secrets"         = "20",
+        "services"        = "20"
+      }
+      manifests_dir = "./manifests2"
+      users = [
+
+        "arn:aws:iam::123456789012:role/other-sso-iam-role"
+      ]
+    }
+  }
+```
+
+The `application_teams` will do the following for every provided team:
+
+- Create a namespace
+- Register quotas
+- Register IAM users for cross-account access
+- Create a shared role for cluster access. Alternatively, an existing role can be supplied.
+- Register provided users/role in the `awsAuth` map for `kubectl` and console access to the cluster and namespace.
+- (Optionally) read all additional manifests (e.g., network policies, OPA policies, others) stored in a provided directory, and applies them.
+
+### PlatformTeam
+
+To create an `Platform Team` for your cluster, simply use `platform_teams`. You will need to supply a team name and and all users/roles.
+
+#### Platform Team Example
+
+```hcl
+  platform_teams = {
+    admin-team-name-example = {
+      users = [
+        "arn:aws:iam::123456789012:user/admin-user",
+        "arn:aws:iam::123456789012:role/org-admin-role"
+      ]
+    }
+  }
+```
+
+`Platform Team` does the following:
+
+- Registers IAM users for admin access to the cluster (`kubectl` and console)
+- Registers an existing role (or create a new role) for cluster access with trust relationship with the provided/created role
+
+## Team Benefits
+
+By managing teams via infrastructure as code, we achieve the following benefits:
+
+1. Self-documenting code
+2. Centralized logic related to the team
+3. Clear place where to add additional provisioning, for example adding Kubernetes Service Accounts and/or infrastructure, such as S3 buckets
+4. IDE support to locate the required team, e.g. CTRL+T in VSCode to lookup class name.
```

Comment:
Can be removed, this is true for the CDK

Reply:
Whoops - removed.